spec/cc/engine/analyzers/java/java_spec.rb

```diff
@@ -142,6 +142,76 @@ module CC::Engine::Analyzers
         issues = run_engine(engine_conf).strip.split("\0")
         expect(issues).to be_empty
       end
+
+      it "prints an issue for similar code when the only difference is the value of two hexadecimal numbers" do
```

Comment:
I like this test a lot, but what do you think about calling this something a bit more generic and simple, like:

`it "prints an issue for similar code" do`

(that would have failed before too right?)